publish-subscribe/src/main/java/com/iluwatar/publish/subscribe/model/WeatherContent.java

```diff
@@ -0,0 +1,18 @@
+package com.iluwatar.publish.subscribe.model;
+
+/** This enum defines the content for {@link Topic} WEATHER. */
+public enum WeatherContent {
+  earthquake("earthquake tsunami warning"),
+  flood("flood start evacuation"),
+  tornado("tornado use storm cellars");
```

Comment:
IMO it's a bit unusual to define constants with an enum. Consider using something like Constants.java along with static final strings.